### PR TITLE
Added OAuth2 support for Github and Google login

### PR DESCRIPTION
**Pull request by brettphillips-perficient**
definitely not production ready ;-)  - but functional for now

### CODE REVIEW

**Review by luatnguyen1 — APPROVED** [review]

**Review by luatnguyen1 — APPROVED** [review]

**Review by luatnguyen1 — APPROVED** [review]

**Review by luatnguyen1 — APPROVED** [review]

File: mtp-app/src/main/resources/application.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  profiles:
+    active: dev             # until another profile is necessary
+
+  security:
+    oauth2:
+      client:
+        registration:
+          github:
+            clientId: ce78b7d31c46a70a11f0
+            clientSecret: cfed7cd48f43b7eef576d51220aa12f329a83aba
```
Comment (vkoul1) [inline mtp-app/src/main/resources/application.yml]: Need to remove these client secrets

**Review by vkoul1 — APPROVED** [review]

File: mtp-app/src/main/resources/application.yml (hunk 1 above)
Comment (rudhrabalankari01) [reply mtp-app/src/main/resources/application.yml]: +1

Comment (brettphillips-perficient) [reply mtp-app/src/main/resources/application.yml]: Done, I thought I already did this but must have pushed it again

**Review by rudhrabalankari01 — APPROVED** [review]
I think that we need to share the credentials internally before merging this. Its better to put a dummy client id and move it to the cloud config server.

**Pull request merged**